src/screens/Login/Login.tsx

```diff
@@ -176,8 +133,9 @@ const Login = () => {
         </TouchableOpacity>
         <Button
           loading={session.signingInStatus === "loading"}
-          onPress={handleOnLogin}
+          onPress={() => handleSubmit()}
```

Comment:
Wouldn't it be better to pass the `handlesubmit` to the onPress like 

` onPress={handleSubmit}`

Reply:
onPress is a GestureRespponse event but handleSubmit is HTMLELement element. there is not mapping for Gesture event currently